env.example

```diff
@@ -1,19 +1,19 @@
-COMPOSE_PROJECT_NAME=io-fn-template
+COMPOSE_PROJECT_NAME=io-fn-backoffice
 
-COSMOSDB_PORT=3000
 API_GATEWAY_PORT=80
```

Comment:
Do we need this?

Reply:
You're right, we don't have an APIM so we can expose the function directly on port 80. I'll remove even `traefik` that is useless on this project.

Reply:
Fixed in 1cc9012